packages/components/input/src/input.css

```diff
@@ -0,0 +1,37 @@
+/* General........................................................ */
+
+:host,
+:host > div {
+	display: block;
+	width: 100%;
+}
+
+::slotted(input) {
+	height: var(--ts-unit-plus);
+	width: 100%;
+	font-size: var(--ts-fontsize-mini);
+	font-weight: var(--ts-fontweight);
+	padding: calc(var(--ts-unit-quarter) - 1px) var(--ts-unit-half);
+	border: var(--ts-border-width) var(--ts-border-type) var(--ts-border-color-default);
+	border-radius: var(--ts-radius);
+	background-color: var(--ts-color-white);
+	white-space: nowrap;
+	word-wrap: normal;
+	overflow: hidden;
+	text-overflow: ellipsis;
+	box-sizing: border-box;
+}
+
+::slotted(input:not([disabled]):hover) {
+	border-color: var(--ts-color-blue-light);
```

Comment:
In Figma design hover state of _ts-input_ borders are 2 pixels thick. Can you check it out, please?

Reply:
Nice catch